api/jwt.go

```diff
@@ -120,17 +121,21 @@ func keyForToken(token *jwt.Token) (interface{}, error) {
 // Encoding RSA pub key: https://play.golang.org/p/mLpOxS-5Fy
 func oathkeeperPublicKeys() (*jwk.Set, error) {
 	if oathkeeperPubKeys == nil {
+		fmt.Println("fetching JWKS from oathkeeper")
```

Comment:
are these print statements left in here intentionally? weren't we using a log library (logrus)?

Reply:
I should transition them to logrus and prune them a bit. But yes, they should not be prints.